test/verify/check-static-login

```diff
@@ -861,6 +861,8 @@ matchrule = <SUBJECT>^DC=LAN,DC=COCKPIT,CN=alice$
         # by IPv6 address and port
         check_server("[::1]:22", True)
 
+        self.allow_failed_sudo_journal_messages()
```

Comment:
This reminds me of #15544 -- it's tempting to just slap on this "ignore sudo", but why? Since we stopped invoking sudo by default, nothing here *should* try to raise privileges?

It also doesn't match the commit message -- this is about sudo, while cockpit/ssh is, well, ssh.

Reply:
> Since we stopped invoking sudo by default, nothing here should try to raise privileges?

This change in default comes only with #15293, which is still open.

Reply:
> It also doesn't match the commit message -- this is about sudo, while cockpit/ssh is, well, ssh.

Hmm, the commit says: `test: Expect sudo failures on remote login`.  Isn't that what you are asking for?  The messages are from sudo, but they happen during remote login.

Reply:
The subject yes, but it seems odd that adding `SYSLOG_IDENTIFIER=cockpit/ssh` would suddenly show sudo errors? So these are sudo messages from the remote machine, which are being forwarded through ssh?

Reply:
No, they are in the journal of the remote machine. They have always been there, but `check_journal_messages` did not retrieve them.